app/commands/GenerateSwagger.php

```diff
@@ -122,15 +122,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         if (!$apiRoutes) {
             $output->writeln("<error>No suitable routes found</error>");
-            return;
+            return 1;
```

Comment:
What is this new devilry?

Reply:
Implementation of `execute` method of `Command` should return integer.

See https://github.com/symfony/console/blob/5.x/Command/Command.php#L163